app/server.R

```diff
@@ -29,9 +29,13 @@ server <- function(input, output, session) {
       updateCheckboxGroupInput(session, "symptoms", selected = character(0))
     }
     # handle special cases (immune and other) in chronic conditions 
-    if ("is_other" %in% input$conditions) {
-      # if other chronic condition is selected, clear all other selections
-      updateCheckboxGroupInput(session, "conditions", selected = "is_other")
+    if ("immunocompromised" %in% input$conditions) {
```

Comment:
Umm I think we fixed the immunocompromised one, so this special treatment should only be applied to "other". Please change comment in line 31 as well.